detectable/src/main/java/com/synopsys/integration/detectable/detectables/maven/cli/MavenCodeLocationPackager.java

```diff
@@ -70,9 +70,9 @@ public MavenCodeLocationPackager(ExternalIdFactory externalIdFactory) {
     }
 
     // mavenTextOutput should be the full output of mvn dependency:tree (no scope applied); scope filtering is now done by this method
-    public List<MavenParseResult> extractCodeLocations(String sourcePath, String mavenOutputText, String excludedScopes, String includedScopes, String excludedModules, String includedModules) {
+    public List<MavenParseResult> extractCodeLocations(String sourcePath, String mavenOutputText, List<String> excludedScopes, List<String> includedScopes, String excludedModules, String includedModules) {
         ExcludedIncludedWildcardFilter modulesFilter = new ExcludedIncludedWildcardFilter(excludedModules, includedModules);
-        ExcludedIncludedWildcardFilter scopeFilter = new ExcludedIncludedWildcardFilter(excludedScopes, includedScopes);
+        ExcludedIncludedWildcardFilter scopeFilter = new ExcludedIncludedWildcardFilter(StringUtils.joinWith(",", excludedScopes.toArray()), StringUtils.joinWith(",", includedScopes.toArray()));
```

Comment:
I know it doesn't currently allow it, but we should be allowed to create ExcludedIncludedWildcardFilter with already created sets/lists. This seems a little silly.

Reply:
Agreed, I just create a PR against integration-common for that change https://github.com/blackducksoftware/integration-common/pull/31